src/qt/transactionview.cpp

```diff
@@ -415,6 +415,9 @@ void TransactionView::bumpFee()
 
     // Bump tx fee over the walletModel
     model->bumpFee(hash);
+
+    // Update the table
+    model->getTransactionTableModel()->updateTransaction(hashQStr, CT_UPDATED, false);
```

Comment:
In commit "Make sure we always update the table row after a bumpfee call"

I was surprised that passing false for the `updateTransaction` `showTransaction` parameter doesn't hide the transaction, though it's good that it doesn't. Do you think this parameter should be removed (separately, not part of this change)? I'm not seeing what it does other than set a temporary state that gets immediately overwritten.